### PR TITLE
add sponsorship tiers and packages

### PR DESCRIPTION
**Pull request by contrepoint**
* monetary and in-kind sponsorship

### CODE REVIEW

File: content/sponsors.md [hunk 1]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
+
+### AWESOME Sponsor! (USD 2,000)
+* Space on a shared table for freebies/swag
+* Roll-up poster or other signage
+* Verbal thanks during the conference
+* Logo on welcome slide
+
+---
+
+## In-Kind Sponsorship
```
Comment (zmagg) [inline content/sponsors.md]: nit: Could we include something here like

## In-Kind Sponsorship (USD variable)

File: content/sponsors.md [hunk 2]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
-available, with more details to come soon; in the mean time, please [contact
-us directly](mailto:bangbangcon.west@gmail.com) if you're considering
-sponsoring the event!
+available. If you have any questions, please [contact
+us.](mailto:bangbangcon.west@gmail.com)
 
+---
+
+## Sponsorship Tiers
+### PHENOMENAL Sponsor!!! (USD 8,000)
```
Comment (zmagg) [inline content/sponsors.md]: could we put a $ sign after usd? so USD $8000

Comment (zmagg) [reply content/sponsors.md]: also, I frequently see denominations with the currency _after_, so it should actually be $8,000 USD

Comment (jwise) [reply content/sponsors.md]: I'm also ok with making this "USD-normative", and just saying somewhere "_All amounts in US dollars._"

(signed,
a stereotypical American)

File: content/sponsors.md [hunk 3]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
+* Space on a shared table for freebies/swag
+* Roll-up poster or other signage
+* Verbal thanks during the conference
+* Logo on welcome slide
+
+---
+
+## In-Kind Sponsorship
+* Accommodation near UCSC
+  * We provide accommodation for our speakers
```
Comment (jwise) [inline content/sponsors.md]: if these are a call for action, they should be aspirational: "We aim to provide accommodation for all of our speakers.  Your business can sponsor us by..."

File: content/sponsors.md [hunk 4]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
+* Live Captioning
+  * In order to make !!Con West as accessible as possible, we have live captioning by a professional stenographer who is familiar with the computing world
```
Comment (jwise) [inline content/sponsors.md]: "we hope to continue our tradition of having live captioning..."

File: content/sponsors.md (hunk 3 above)
Comment (zmagg) [reply content/sponsors.md]: How about:

We aim to provide accommodation for all of our speakers. You could sponsor accommodation for all speakers this year!

File: content/sponsors.md (hunk 4 above)
Comment (zmagg) [reply content/sponsors.md]: How about something like:

We hope to continue our tradition of having live captioning, a practice !!Con helped make more mainstream. You could help sponsor our ability to have live captioning by a professional stenographer!

File: content/sponsors.md [hunk 5]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
+* Child Care
+  * In order to make !!Con West as accessible as possible, we would like to provide child care for our attendees
```
Comment (zmagg) [inline content/sponsors.md]: How about instead:

You could help us sponsor child care for our attendees, making !!Con West accessible and inclusive to more people.

File: content/sponsors.md (hunk 2 above)
Comment (contrepoint) [reply content/sponsors.md]: @zmagg good catch - I googled to check. For USD, it seems to be `X USD` or `$ X` so I'm going with `X USD` to be clear on currency.
@jwise In the spirit of making things as inclusive as possible, I'd rather not make this USD-normative.

File: content/sponsors.md [hunk 6]
```diff
@@ -18,7 +18,42 @@ remains affordable to everybody who wants to participate.
...
+* A staffable booth with space for swag, company information
+* 3 reserved tickets for employees*
+* Roll-up poster or other signage
+* Verbal thanks during the conference
+* Logo on welcome slide
+* Logo on website
+
+### EXCELLENT Sponsor!! (5,000 USD)
+* Space on a shared table for freebies/swag
+* 1 reserved ticket for employees*
```
Comment (De-ma) [inline content/sponsors.md]: The asterisk was something !!Con nyc sponsorships page included that says at the end of the page 

"A note about employee tickets
Space at !!Con is limited, so we humbly request that these tickets go to employees of your company who will really enjoy the subject matter of the talks. What that means will depend on the company, but it’s likely to mean programmer/engineer employees or programming enthusiasts."

Do we want to add that as well?

**Comment by jwise** [comment]
merged as #20

**Pull request closed without merging**